modules/swagger-core/src/test/java/io/swagger/jackson/XMLInfoTest.java

```diff
@@ -34,14 +35,19 @@ public void testSimple() throws Exception {
         assertNotNull(xml);
         assertEquals(xml.getName(), "xmlDecoratedBean");
 
-        final Property property = impl.getProperties().get("items");
+        // Cast it to an array property
+        final ArrayProperty property = (ArrayProperty)impl.getProperties().get("items");
         assertNotNull(property);
         final Xml propertyXml = property.getXml();
-
         assertNotNull(propertyXml);
-        assertEquals(propertyXml.getName(), "item");
+        // Check the name of array property
+        assertEquals(propertyXml.getName(), "items");
         assertTrue(propertyXml.getWrapped());
-
+        // Get the xml for items for the array property
+        final Xml itemsXml = property.getItems().getXml();
```

Comment:
I agree with both the points. I did not change them earlier because they were part of the existing code. Have made changes now. Thanks!